als-lsp/js/src/main/scala/org/mulesoft/lsp/convert/LspConvertersClientToShared.scala

```diff
@@ -487,11 +487,21 @@ object LspConvertersClientToShared {
 
   implicit class DidChangeConfigurationNotificationParamsConverter(v: ClientDidChangeConfigurationNotificationParams) {
     def toShared: DidChangeConfigurationNotificationParams =
-      DidChangeConfigurationNotificationParams(v.mainUri,
-                                               v.folder.toOption,
-                                               v.dependencies.toSet,
-                                               v.customValidationProfiles.toSet,
-                                               v.semanticExtensions.toSet)
+      DidChangeConfigurationNotificationParams(
+        v.mainUri,
+        v.folder.toOption,
+        v.dependencies.map { d =>
+          d.asInstanceOf[Any] match {
+            case s: String => Left(s)
+            case _         => Right(d.asInstanceOf[ClientDependencyConfiguration].toShared)
+          }
+        }.toSet
+      )
+  }
+
+  implicit class DependencyConfigurationConverter(v: ClientDependencyConfiguration) {
+    def toShared: DependencyConfiguration =
+      DependencyConfiguration(v.file, v.scope)
```

Comment:
Could be talk about the v.file field?. Instead of point to a path of a file (or some resource), could be have there a "ConfigurationParams"? I mean, a complete project configuration?

Reply:
https://www.mulesoft.org/jira/browse/ALS-1724